src/appendix/stupid-stats.md

```diff
@@ -1,9 +1,12 @@
 # Appendix A: A tutorial on creating a drop-in replacement for rustc
 
 > **Note:** This is a copy of `@nrc`'s amazing [stupid-stats]. You should find
-> a copy of the code on the GitHub repository although due to the compiler's
-> constantly evolving nature, there is no guarantee it'll compile on the first
-> go.
+> a copy of the code on the GitHub repository.
+>
+> Due to the compiler's constantly evolving nature, the `rustc_driver`
+> mechanisms described in this chapter have been replaced by a new
```

Comment:
Not exactly true as `rustc_driver` still exposes a similar interface with `run_compiler` + `Callbacks`.